classes/recording.php

```diff
@@ -607,8 +607,8 @@ protected function metadata_get($fieldname) {
     protected static function fetch_records(array $selects, array $params): array {
         global $DB;
 
-        // Fetch the local data.
-        $recordings = $DB->get_records_select(static::TABLE, implode(" AND ", $selects), $params);
+        // Fetch the local data. Arbitrary sort by id, so we get the same result on different db engines.
+        $recordings = $DB->get_records_select(static::TABLE, implode(" AND ", $selects), $params, 'id ASC');
```

Comment:
Suggest the timecreated desc rather than id asc. This will just lead to bug reports where people get their recordings in weird orders on some setups.

Reply:
Yes, that's true. Was not sure about which column to take. But it seem not to resolve the problem ref. the CI tests though.